src/steps/05-PerformanceRequirements/DOW/OtherOfferings.vue

```diff
@@ -76,17 +76,32 @@
     <v-form ref="form" lazy-validation>
 
       <div v-if="selectedClassificationLevelList.length > 1" class="mb-10">
-        <ATATRadioGroup
-          id="ClassificationLevel"
-          legend="What classification level is this instance deployed in?"
-          :value.sync="_serviceOfferingData.classificationLevel"
-          :items="classificationRadioOptions"
-          name="ClassificationLevel"
-          class="mt-3 mb-2"
-          :tooltipText="classificationTooltipText"
-          tooltipLabel="Classification level for this instance"
-          :rules="[$validators.required('Please select a classification level.')]"
-        />
+        <div>
+          <ATATCheckboxGroup
+            v-if="isPortabilityPlan"
+            id="ClassificationLevel"
+            groupLabel="What classification level is this instance deployed in?"
```

Comment:
Portability Plan groupLabel should be `What classification level(s) do you need a Portability Plan for?`